code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -336,7 +336,9 @@ var/total_borer_hosts_needed = 10
 		if(H!=src && Adjacent(H))
 			choices += H
 
-	var/mob/living/carbon/H = input(src,"Who do you wish to infest?") in null|choices
+	if(!choices.len)
+		return
+	var/mob/living/carbon/H = choices.len > 1 ? input(src,"Who do you wish to infest?") in null|choices : choices[1]
```

Comment:
if ```choices.len``` is 2 or more, we go to the input.
if it's 1, we go to ```choices[1]```
if it's 0... we go to ```choices[1]``` which doesn't exist, @CYBERBOSS  !!!